src/jsutils/nodejsCustomInspectSymbol.d.ts

```diff
@@ -0,0 +1,4 @@
+// istanbul ignore next (See: https://github.com/graphql/graphql-js/issues/2317)
```

Comment:
Why did you code `Istanbul` comment into `d.ts` file it's not part of a coverage?